app/config/init.js

```diff
@@ -36,6 +36,8 @@ const _init = function (cfg) {
       notify('Error reading configuration: `config` key is missing');
       return _extractDefault(cfg.defaultCfg);
     }
+    // Ignore undefined values in plugin array Issue #1862
+    _cfg.plugins = _cfg.plugins.filter(Boolean);
```

Comment:
If `localPlugins`is empty, it will do nothing. It is pretty useless you're right, but it is better for consistency imo.